activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+*   `ActiveRecord::Encryption::Encryptor` now supports a `:compressor` option to customize the compression algorithm used.
+
+    ```ruby
+    module ZstdCompressor
+      def self.inflate(data)
+        Zstd.decompress(data)
+      end
+
+      def self.deflate(data)
```

Comment:
Like above but `compress`

Reply:
@pokonski Thank you! I fixed it.

Reply:
It looks like the changelog still shows `inflate/deflate`

Reply:
@zzak I made a mistake while rebasing 😓.